samples/swiftui/FirebaseSwiftUIExample/FirebaseSwiftUIExample/App/ContentView.swift

```diff
@@ -0,0 +1,175 @@
+    actionCodeSettings.setIOSBundleID(Bundle.main.bundleIdentifier!)
+    let configuration = AuthConfiguration(
+      tosUrl: URL(string: "https://example.com/tos"),
+      privacyPolicyUrl: URL(string: "https://example.com/privacy"),
+      emailLinkSignInActionCodeSettings: actionCodeSettings,
+      mfaEnabled: true
+    )
+    
+    // Create Facebook provider with Limited Login enabled by default
+    let fbProvider = FacebookProviderSwift()
+    fbProvider.setLimitedLogin(true)
+    facebookProvider = fbProvider
+    
+    authService = AuthService(
+      configuration: configuration
+    )
+    .withAppleSignIn()
+    .withPhoneSignIn()
+    .withGoogleSignIn()
+    .withFacebookSignIn(facebookProvider)
+    .withTwitterSignIn()
+    .withOAuthSignIn(OAuthProviderSwift.github())
+    .withOAuthSignIn(OAuthProviderSwift.microsoft())
+    .withOAuthSignIn(OAuthProviderSwift.yahoo())
+    .withEmailSignIn()
+  }
+  
+  let authService: AuthService
+  let facebookProvider: FacebookProviderSwift
+  // State for Facebook Limited Login toggle
+  @State private var useLimitedLogin = true
+  @State private var isPresented = false
+  
+  var body: some View {
+    VStack {
+      AuthPickerView(
+        isPresented: $isPresented,
+        interactiveDismissDisabled: true
+      ) {
+        NavigationStack {
+          VStack {
+            if authService.authenticationState == .unauthenticated {
+              Text("Not Authenticated")
+            } else {
+              Text("Authenticated - \(authService.currentUser?.email ?? "")")
+              Button {
+                Task {
+                  try? await authService.signOut()
+                }
+              } label: {
+                Text("Sign Out")
+              }
+              .buttonStyle(.borderedProminent)
+            }
+          }
+          .navigationTitle("Firebase UI Demo")
+        }
+        .onAppear {
+          isPresented = authService.authenticationState == .unauthenticated
+        }
+        .onChange(of: authService.authenticationState) { oldValue, newValue in
+          debugPrint("authService.authenticationState - \(newValue)")
+          if newValue != .authenticating {
+            isPresented = newValue == .unauthenticated
+          }
+        }
+      } footer: {
+        // Facebook Limited Login Control (Example)
+        GroupBox {
+          VStack(alignment: .leading, spacing: 8) {
+            Text("Facebook Settings")
+              .font(.headline)
+            
+            Toggle("Use Limited Login", isOn: $useLimitedLogin)
+              .onChange(of: useLimitedLogin) { _, newValue in
+                handleLimitedLoginToggle(newValue)
+              }
+            
+            Text("Limited Login protects privacy by preventing Facebook tracking across apps.\nNOTE: THIS IS NOT PART OF THE SDK")
+              .font(.caption)
+              .foregroundColor(.secondary)
+          }
+          .padding()
+        }
+      }
```

Comment:
This should all be removed as mentioned above. Should just be `AuthPickerView().environment(authService)`.